src/content/docs/security/security-privacy/compliance/fedramp-compliant-endpoints.mdx

```diff
@@ -419,12 +419,84 @@ Follow these instructions as applicable to set the FedRAMP endpoint for a specif
 
 ## OTLP API [#otlp-api]
 
-To ensure FedRAMP compliance when using the [OpenTelemetry API](/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start#review-settings):
+To ensure FedRAMP compliance when using the OpenTelemetry API, only send data to the following endpoints based on the data type and protocol:
 
-* Do not send data to the US OTLP API endpoint of `https://otlp.nr-data.net:4317`.
-* Instead, send data to `https://gov-otlp.nr-data.net:4317`. Or, if your client does not support gRPC, you may use port 4318 for OTLP/HTTP.
+    <table>
+      <thead>
+        <tr>
+          <th>
+            Data type
+          </th>
+          <th>
+            Protocol
+          </th>
+          <th>
+            Port
+          </th>
+          <th>
+            URL
+          </th>
+        </tr>
+      </thead>
+        <tbody>
+        <tr>
+          <td>
+            Traces and logs
+          </td>
+          <td>
+            gRPC
+          </td>
+          <td>
+            4317, 443
+          </td>
+          <td>
+            `https://gov-otlp.nr-data.net:4317`
+
+            `https://gov-otlp.nr-data.net:443`
+
+            We don't recommend that you use the gRPC protocol at this time because the endpoint only accepts traces and logs. This is due to technical limitations in the OTLP metrics integration. We recommend you send your all your OTLP FedRamp data (metrics, traces, and logs) via the HTTP protocol. If your application doesn't support HTTP, you can still just send traces and logs via gRPC (not metrics).
```

Comment:
```suggestion
            We don't recommend that you use the gRPC protocol at this time because the endpoint only accepts traces and logs. This is due to technical limitations in the OTLP metrics integration. We recommend you send all your OTLP FedRamp data (metrics, traces, and logs) via the HTTP protocol. If your application doesn't support HTTP, you can still just send traces and logs via gRPC (not metrics).
```